src/pages/PatientsPage.tsx

```diff
@@ -11,18 +11,23 @@ const PatientsPage = (): JSX.Element => {
 
   return (
     <>
-      <div style={{ display: 'flex', alignItems: 'center' }}>
-        <h1>{t('title.myPatients')}</h1>
-        <Button variant='primary' className='ms-auto' href='/patients/create'>
+      <div className='position-fixed bottom-0 end-0 me-3 mb-5 pb-5' style={{ zIndex: '1030' }}>
+        <Button variant='info' href='/patients/create'>
```

Comment:
This button deserves its own component since it's used somewhere else in the preview PR